samples/CommunityToolkit.Maui.Sample/CommunityToolkit.Maui.Sample.csproj

```
@@ -64,16 +64,16 @@
     <!-- Custom Fonts -->
     <MauiFont Include="Resources\Fonts\*" />
 
-    <PackageReference Include="Microsoft.Maui.Controls" Version="*" />
+    <PackageReference Include="Microsoft.Maui.Controls" Version="$(MauiPackageVersion)" />
```

Reminder to set this back to `*` to always use the latest non-preview version of `Microsoft.Maui.Controls`

```suggestion
    <PackageReference Include="Microsoft.Maui.Controls" Version="*" />
```